### PR TITLE
Fixed some grammatical errors in README.md

### PR DESCRIPTION
**Pull request by JanumalaAkhilendra**
Description:
This Pull Request fixes a grammatical error in the README.md file.

Changes Made:
Fixed some grammatical errors in README.md

Additional Information:
This fix is not related to any existing issue. It's a minor typo that I noticed while reviewing the README.

Thank you for considering this contribution.

### CODE REVIEW

**Review by DzikuVx — CHANGES_REQUESTED** [review]
please resolve conflicts

File: README.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ On some Linux distros, you may be missing `libatomic`, a `NW.JS` (specially `lib
 * Debian / Ubuntu: `sudo apt install libatomic1`
 * Fedora: `sudo dnf install libatomic`
 
-1. Don't forget to add your user into dialout group "sudo usermod -aG dialout YOUR_USERNAME" for serial access
```
Comment (MrD-RC) [inline README.md:87]: This needs to stay as dialout.

File: README.md [hunk 2]
```diff
@@ -113,22 +113,22 @@ in the configurator. If new files are added, they must be included in `gulpfile.
...
+directory. Running this task on macOS or Linux requires Wine since it's needed to set the icon
 for the Windows app. If you don't have Wine installed you can create a release by running the **release-only-linux** task.
-<br>`--installer` argument can be added to build installers for particular OS. NOTE: MacOS Installer can be built with MacOS only.
+<br>`--installer` argument can be added to build installers for a particular OS. NOTE: MacOS Installer can be built with MacOS only.
 
 To build a specific release, use the command `release --platform="win64"` for example.
 
 ### Running with debug | Inspector
 
-To be able to open Inspector, you will need SDK flavours of NW.js
```
Comment (MrD-RC) [inline README.md:124]: This is not spelt incorrectly

File: README.md [hunk 3]
```diff
@@ -145,22 +145,22 @@ INAV Configurator is shipped **WITHOUT** API key for Bing Maps. That means: ever
...
 	1. Enter MapProxy service layer (inav_layer if configured from MAPPROXY.md)
-1. Once completed, you can zoom in on area you will be flying in while connected to the internet in either GPS or Mission Control tab to save the cache for offline use
+1. Once completed, you can zoom in on the area you will be flying in while connected to the internet in either GPS or Mission Control tab to save the cache for offline use
 
 ## Font Customisation
 
-INAV provides the font images so that custom fonts can be created for your personal preference. This is the case for both analogue and digital fonts. The resources can be found in the [osd](/resources/osd) folder. Within the **analogue** and **digital** subfolders, you will find information on compiling your own fonts. There is also an [INAV Character Map](/resources/osd/INAV%20Character%20Map.md) document. This contains previews of all the character images in the fonts, and the appropriate variable names within the firmware and Configurator. There are tools for compiling the [analogue](https://github.com/fiam/max7456tool) and [digital](https://github.com/MrD-RC/hdosd-font-tool) fonts. New font submissions via pull requests are welcome.
```
Comment (MrD-RC) [inline README.md:163]: Analogue is not spelt incorrectly

**Review by MrD-RC — CHANGES_REQUESTED** [review]

**Comment by MrD-RC** [comment]
I’m sure these have already been changed. I merged a PR a few days ago which had these changes.

**Comment by MrD-RC** [comment]
#1857 already resolved these.

**Pull request closed without merging**